src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/EnrollmentEditEventPage.component.js

```diff
@@ -73,6 +76,11 @@ export const EnrollmentEditEventPageComponent = ({
             pageLayout={pageLayout}
             currentPage={mode === EnrollmentPageKeys.EDIT_EVENT ? EnrollmentPageKeys.EDIT_EVENT : EnrollmentPageKeys.VIEW_EVENT}
             availableWidgets={WidgetsForEnrollmentEventEdit}
+            userInteractionInProgress={mode === EnrollmentPageKeys.EDIT_EVENT}
```

Comment:
We should probably refactor this at some point, but just following the logic in `src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/TopBar.container.js`